sdk/core/core-http/test/credentialTests.ts

```diff
@@ -13,29 +14,25 @@ const dummyPassword = "IL0veDummies";
 describe("Basic Authentication credentials", () => {
   const encodedCredentials = base64.encodeString(dummyUsername + ":" + dummyPassword);
   describe("usage", () => {
-    it("should base64 encode the username and password and set auth header with baisc scheme in request", (done) => {
+    it("should base64 encode the username and password and set auth header with basic scheme in request", async () => {
```

Comment:
I thought the test guidance said to not use arrow functions with mocha?

Reply:
I'll update it! Thanks!

Reply:
I believe that addressing the test guidelines should probably be done as part of another PR. If we want this change you mention here, should I do it in all of the other test cases? A separate PR could be cleaner for that purpose.

Reply:
yeah I'm good with a second PR

Reply:
I'll be making issues related to these polishes after the test guidelines PR is merged: https://github.com/Azure/azure-sdk-for-js/pull/7784

@willmtemple I want to work with you making new ESLint functionalities to help us enforce the test guidelines :0 that would be amazing. I want to do them myself, but I want to loop you in to at least help me figure out my way through these :)